angelsdev-unit-test/python/unit_test_configuration.py

```diff
@@ -328,6 +369,7 @@ def addConfiguration(cls:UnitTestConfiguration, configName:str, modList:list[str
       "angels-enable-industries": False, # Angels overhaul
       "angels-enable-components": False, # Angels component mode
       "angels-enable-tech"      : False, # Angels technology mode
+      "angels-enable-acids"     : True   # Restore setting back to default
```

Comment:
This is not how default values work...
```suggestion
```

At line 32, you should add the following line instead:
```lua
# Angels refining
UnitTestConfiguration.addDefaultSetting("startup", "angels-enable-acids", True)
```